frontend/public/locales/en/learn/retirement-income-sources.json

```diff
@@ -0,0 +1,174 @@
+      "header": "Canada Pension Plan (CPP) Program"
+    },
+    "header": "Learn More",
+    "old-age-security-program": {
+      "description": "The official Canada.ca page of the OAS program with information on the OAS retirement pension, and other OAS related benefits.",
+      "header": "Old Age Security Program"
+    },
+    "sources-of-retirement-income": {
+      "description": "Public pensions, OAS, CPP, employer pensions, RRSPs and other sources of personal savings.= from the Financial Consumer Agency of Canada",
+      "header": "Sources of retirement income"
+    },
+    "transitioning-from-work-to-retirement": {
+      "description": "You can take many different paths transitioning from work to retirement. Learn how you can better combine your income from job and pensions.",
+      "header": "Transitioning from work to retirement"
+    }
+  },
+  "old-age-security-program": {
+    "guaranteed-income-supplement": {
+      "header": "Guaranteed Income Supplement (GIS)",
+      "list": {
+        "income-below-maximum": {
+          "content": "your income is <a>below the maximum annual income threshold for the GIS based on your marital status</a>.",
+          "link": "https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/guaranteed-income-supplement/benefit-amount.html"
+        },
+        "live-canada": "live in Canada",
+        "receive-pension": "receive the OAS pension"
+      },
+      "paragraph-1": "The GIS will support you, if you receive the OAS pension and have low income. To qualify for the GIS, you must:",
+      "paragraph-2": "The amount of GIS you can get depends on your income in the previous year and marital status. If you delay the OAS pension, you cannot receive the GIS until you start receiving the OAS pension.",
+      "paragraph-3": {
+        "content": "To learn more, visit the <a>GIS eligibility page</a>.",
+        "link": "https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/guaranteed-income-supplement.html#gc-document-nav"
+      },
+      "paragraph-4": {
+        "content": "To estimate the amount of GIS you may get, visit the <a>Old Age Security benefits estimator</a>",
+        "link": "https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/payments.html"
+      }
+    },
+    "header": "Old Age Security (OAS) program",
+    "oas-pension": {
+      "card": {
+        "content": "The OAS and CPP provide basic inflation-proof retirement income. Lean how you can increase your monthly and life-time pensions.",
+        "header": "When to take your government pensions",
+        "learn-more": "Learn More"
+      },
+      "header": "OAS pension",
+      "paragraph-1": {
+        "content": "The OAS pension is a monthly payment paid to eligible seniors.<br />To see if you qualify for the OAS pension, visit <a>this OAS Pension page</a>.",
```

Comment:
This `<br/>` seems out of place. No other content here seems to have it. Was it intentional?